Firestore/core/test/firebase/firestore/auth/firebase_credentials_provider_test.mm

```diff
@@ -65,12 +65,14 @@ void SetUp() override {
 
   FirebaseCredentialsProvider credentials_provider([FIRApp defaultApp]);
   credentials_provider.GetToken(
-      /*force_refresh=*/true, [](Token token, const absl::string_view error) {
+      /*force_refresh=*/true, [](Token token, const int64_t error_code,
```

Comment:
see above. maybe s/true/kForceRefresh/ ?

Reply:
This is consistent with the other clients and correct as far as that goes. However we're not using it consistently within this PR and should.